src/services/YotiCallbackProcessor.ts

```diff
@@ -166,6 +171,9 @@ export class YotiCallbackProcessor {
   					f2fSession.clientIpAddress,
   					absoluteTimeNow,
   				),
+					document_details: documentFields,
+					gpg45_score: evidence,
+					contra_indicators: evidence[0].ci
```

Comment:
just to check - has this spec been agreed with the TXMA team?

Reply:
We haven't agreed on exact spec with TxMA but were asked to include the above from TICF